src/main/java/org/geppetto/frontend/controllers/WebsocketConnection.java

```diff
@@ -205,6 +223,20 @@ public void onMessage(String message) {
 					connectionHandler.checkUserPrivileges(requestID);
 					break;
 				}
+				case RECONNECT:
+				{
+					parameters = new Gson().fromJson(gmsg.data, new TypeToken<HashMap<String, String>>()
+					{
+					}.getType());
+					String lostConnectionID = parameters.get("connectionID");
```

Comment:
@ddelpiano is this the same value coming as `clientID` after the first connection?
Example from vfb:
`{"type":"client_id","data":"{\"clientID\":\"Connection723\"}"}`